gulpfile.js

```diff
@@ -0,0 +1,66 @@
+const gulp = require('gulp')
```

Comment:
This gulpfile is responsible for pushing the static files created by `nuxt generate` to S3 and then invalidating the correlating files in the cloudfront distribution. I found it here: https://nuxtjs.org/faq/deployment-aws-s3-cloudfront/